### DISCUSSION TITLE
How to reduce file size of Pupil Capture recordings?

### DISCUSSION THREAD

**Discussion by kjkpoi** [pupil-labs/pupil#2290] [Q&A] [answered]
In my pupil recording, it takes more than 1 hours.

Pupil video size is almost 12~16 GB. it makes me harder to use player with this large file.

There are many times program dies by memory leak.

When I record, is there any option to change record encoding?

**Comment by cpicanco** [comment]
Unless you have a high spec computer, I would recommend recording sessions with up to 20-30 min. 

There is also an option in pupil capture to reduce the size of recordings while sacrificing some processing power. However, as far as I known, this option is intended for saving space, not for improving pupil player performance.

**Comment by kjkpoi** [comment]
@cpicanco Could you recommend the option for saving space?

**Comment by cpicanco** [comment, marked as answer]
@kjkpoi you can find the option in pupil capture->recorder->compression:

![screenshot from 2018-03-12 16-31-53](https://user-images.githubusercontent.com/6511849/37305396-10866edc-2613-11e8-8298-e3159981be9f.png)
